src/dredd-command.coffee

```diff
@@ -266,7 +269,6 @@ class DreddCommand
       , waitMilis
 
   run: ->
-    logger.verbose('Preparing for start of testing.')
```

Comment:
This logging happens before logging level is set, so it can never make it to the output.